Guardian/API/API.swift

```diff
@@ -108,4 +108,27 @@ public protocol API {
      - returns: a DeviceAPI instance
      */
     func device(forEnrollmentId id: String, token: String) -> DeviceAPI
```

Comment:
I've marked this one as deprecated (in a separate PR) in case you want to do the same. While recently issued opaque tokens would still work with this method, the other one should cover all scenarios.

Reply:
Thanks, deprecated it in https://github.com/auth0/Guardian.swift/pull/91/commits/53448534fced8b4c19cd9172898010d8b62bf4b2.